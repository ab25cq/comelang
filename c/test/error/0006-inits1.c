/*
PATTERN: size
PATTERN: c:6:
*/

int x[3] = {1, 2};

