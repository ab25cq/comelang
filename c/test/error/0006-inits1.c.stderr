array initializer wrong size for type at test/error/0006-inits1.c:6:12
int x[3] = {1, 2};
           ^
